sample/Droidcon17/config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "Email": "dev@fossasia.org",
+  "App_Name": "Droidcon",
```

Comment:
I think it should be DroidCon instead of Droidcon everyhwere to match the title.